applications/daily_tests/.eslintrc.json

```diff
@@ -0,0 +1,36 @@
+{
+  "extends": ["eslint:recommended", "plugin:prettier/recommended"],
+  "plugins": ["prettier", "@babel"],
+  "env": {
+    "browser": true,
+    "commonjs": true,
+    "node": true,
+    "es2021": true
+  },
+  "parser": "@babel/eslint-parser", // Tell ESLint that you want to use the @babel/eslint-parser.
```

Comment:
lol I thought comments weren't allowed in json

Reply:
interesting https://github.com/mozilla/addons-frontend/issues/942

Reply:
Yeah also saw that, very needed - basically the worst thing about JSON as config